cypress/locators/date-input/index.js

```diff
@@ -11,6 +11,7 @@ export const maxDate = () => cy.get(MAX_DATE);
 
 // component preview locators
 export const dateInput = () => cy.iFrame(DATE_INPUT);
```

Comment:
do we need those other `inIFrame` locators ?

Reply:
yes, we need it. We have TC which is opened in Iframe state and use it